docs/docs/recipes/working-with-images.md

```diff
@@ -346,24 +350,26 @@ exports.createPages = async ({ graphql, actions }) => {
 }
 ```
 
-3. Now, import `Img` from `gatsby-image`, and `graphql` from `gatsby` into the template component, write a [pageQuery](/docs/page-query/) to get image data based on the passed in `slug` and pass that data to the `<Img />` component:
+3. Maintenant, importez `Img` de `gatsby-image` et `graphql` de `gatsby` dans le composant de modèle, écrivez une [pageQuery](/docs/page-query/) pour obtenir des données d'image basées sur les dans `slug` et transmettez ces données au composant` <Img /> `:
 
 ```jsx:title=markdown-layout.jsx
 import React from "react"
 import { graphql } from "gatsby" // highlight-line
 import Img from "gatsby-image" // highlight-line
 
-export default ({ children, data }) => (
-  <main>
-    // highlight-start
-    <Img
-      fluid={data.markdown.frontmatter.image.childImageSharp.fluid}
-      alt="A corgi smiling happily"
-    />
-    // highlight-end
-    {children}
-  </main>
-)
+export default function Layout({ children, data }) {
```

Comment:
Please don't change functions

Reply:
export default function Layout({ children, data }) {
  return (
    <main>
      // highlight-start
      <Img
        fluid={data.markdown.frontmatter.image.childImageSharp.fluid}
        alt="A corgi smiling happily"
      />
      // highlight-end
      {children}
    </main>
  )
}

I think there is no change i can see here, could you let me know what is issue here exactly

Reply:
Le code a été changé après la ligne 356, merci de remettre l'ancien